x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -536,19 +551,34 @@ export const ActionForm = ({
         </SectionLoading>
       ) : (
         <Fragment>
-          {alertActionsList}
-          <EuiSpacer size="m" />
-          {isAddActionPanelOpen === false ? (
-            <EuiButton
-              iconType="plusInCircle"
-              data-test-subj="addAlertActionButton"
-              onClick={() => setIsAddActionPanelOpen(true)}
-            >
+          <EuiTitle size="s">
+            <h4>
               <FormattedMessage
-                id="xpack.triggersActionsUI.sections.alertForm.addActionButtonLabel"
-                defaultMessage="Add action"
+                defaultMessage="Actions"
+                id="xpack.triggersActionsUI.sections.alertForm.actionSectionsTitle"
               />
-            </EuiButton>
+            </h4>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          {alertActionsList}
+          {isAddActionPanelOpen === false ? (
+            <div>
+              <EuiHorizontalRule />
+              <EuiFlexGroup>
+                <EuiFlexItem grow={false}>
+                  <EuiButton
```

Comment:
This "Add action" button shows on the left for me, not sure if you wanted it moved to the right?

<img width="606" alt="Screen Shot 2020-04-15 at 11 04 19 AM" src="https://user-images.githubusercontent.com/3694571/79355365-8d499900-7f0b-11ea-96c5-dbb820e3d871.png">

Reply:
Are we using "Add new" on other screens"?  Better wording would be something specific, like "Add action" or "Add connector"

Reply:
> This "Add action" button shows on the left for me, not sure if you wanted it moved to the right?
> 
> <img alt="Screen Shot 2020-04-15 at 11 04 19 AM" width="606" src="https://user-images.githubusercontent.com/3694571/79355365-8d499900-7f0b-11ea-96c5-dbb820e3d871.png">

Yes, we moved this button to the left as we wanted to have the `Save` button be the only thing on the right.

Reply:
Oh I see, thanks!